### DISCUSSION TITLE
Thảo luận tuần 2 + lên ideas cho bài làm

### DISCUSSION THREAD

**Discussion by GiaHan06** [GiaHan06/Lop-10A02-Nhom-02#19] [Ideas]
mại zô mại zô

**Comment by NguyenNgocQuynhAnh** [comment]
kkk

**Reply by GiaHan06** [reply]
hehehe

**Comment by GiaHan06** [comment]
hù

**Reply by LTGHan** [reply]
Kkk

**Reply by chaudambaotram** [reply]
hế nhô

**Comment by GiaHan06** [comment]
h sao:> có ai nghĩ được gì ckua

**Reply by chaudambaotram** [reply]
t hổng nghĩ ra đc j lun ấy  :)

**Comment by LTGHan** [comment]
Hmmm

**Reply by GiaHan06** [reply]
sao chời :'(

**Comment by GiaHan06** [comment]
nội dung nè
"Sử dụng thành ngữ / tục ngữ / ca dao hay thơ tự sáng tác để bày tỏ quan điểm, nghị luận về cuộc sống xung quanh mà bản thân cảm nhận được"

**Reply by LTGHan** [reply]
Theo tui hỉu í là mình làm 1 bài văn nói về quan điểm của mình về cuộc sống xung quanh thông qua các thành ngữ cao dao tục ngữ

**Reply by GiaHan06** [reply]
maybe
tui cũng có suy nghĩ giống zạy

**Reply by LTGHan** [reply]
Nếu z trc tiên mình cần lập 1 dàn ý ra coi mình sẽ viết những nội dung gì trước nhỉ

**Reply by GiaHan06** [reply]
mà chắc là 3 cái câu đó mình làm chung trong một bài luôn chứ hong tách ra riêng ha?

**Comment by HuynhLeBaoTran** [comment]
í là giờ mình phân công việc thui đk

**Reply by GiaHan06** [reply]
bh mình cần phải phân tích cái đề xong ròi đưa ra hướng giải quyết sau đó phân công cụ thể nv nha bà

**Comment by GiaHan06** [comment]
http://baoninhthuan.com.vn/news/90611p0c77/lang-mot-chut-de-cam-nhan-cuoc-song.htm
cái này nó hong liên quan lắm nhma có thể cho tụi mình xíu ý tưởng
hong biết nữa mà mấy bà đọc thử đi

**Reply by GiaHan06** [reply]
http://baoninhthuan.com.vn/news/90611p0c77/lang-mot-chut-de-cam-nhan-cuoc-song.htm
nữa nè

**Reply by GiaHan06** [reply]
chỉ mang tính chất tham khảo nha

**Comment by NguyenNgocQuynhAnh** [comment]
helu đến  đâu rồi mấy bà

**Reply by GiaHan06** [reply]
bh đang tìm hướng giải quyết đề nè bà

**Comment by GiaHan06** [comment]
hay bh mình lên dàn ý trước cho dễ, từ đó mình đi sâu hơn
@LTGHan có ý gì hong nè
@chaudambaotram @NguyenNgocQuynhAnh @HuynhLeBaoTran mấy bà cũng nghĩ luôn nhe

**Reply by LTGHan** [reply]
Nói dìa co vi hôk

**Reply by GiaHan06** [reply]
nói tới những khó khăn xong lạng qua tình cảm con người?

**Reply by LTGHan** [reply]
Adu lụa đó

**Reply by chaudambaotram** [reply]
công nhận

**Reply by GiaHan06** [reply]
> Nói dìa co vi hôk

ý em muốn là gì, ý em muốn là sao

**Reply by LTGHan** [reply]
> 

Z i

**Reply by GiaHan06** [reply]
trời ơi quả nhiên là Lưu Hân:> gợi ý cái ra liền

**Reply by GiaHan06** [reply]
thêm 1 cái zì cho nó đặc sắc nữa hong hay còn gì liên quan nữa, khai thác nhiều zô

**Reply by chaudambaotram** [reply]
t thì mún làm về cuộc sống học đường nhưng mà để tìm câu ca dao tực ngữ khó quá

**Reply by GiaHan06** [reply]
hmmmm đúng là tìm khó thiệt, t cũng tìm quá trời nhma toàn kiểu nói về mấy cái đức tính hay gì ấy chứ cũng hong liên quan

**Reply by HuynhLeBaoTran** [reply]
làm về covi như lhan nói đựt á

**Reply by HuynhLeBaoTran** [reply]
nhìu thơ để tìm

**Reply by NguyenNgocQuynhAnh** [reply]
chắc zề covid là ô kê nhứt đóa

**Reply by GiaHan06** [reply]
zạy cũng được nhma mình phải khai thác nhiều ý trong cái covi đó ra, mình cảm nhận được những gì mình cứ nói ra hết á xong có gì sửa sau
hmmm tui cũm hong biết nói sao nữa mà mấy bà hiểu ý tui hong zị

**Reply by GiaHan06** [reply]
> nói tới những khó khăn xong lạng qua tình cảm con người?

nếu mà chốt covi thì chắc làm như zày được ha, nhma thầy bảo 3 câu ca dao tục ngữ mà này 1 cái về khó khăn 1 cái về tình cảm con người được 2 còn 1 nữa thì sao

**Reply by chaudambaotram** [reply]
1 câu nữa để làm phần mở bài hay kết bài nói chung lại cuộc sống covi và con người ấy.

**Comment by GiaHan06** [comment]
ai có ý tưởng thì cứ việc cmt nha

**Reply by NguyenNgocQuynhAnh** [reply]
kiểu kiếm bài thơ nói về cuộc sống sau khi covid được hạn chế hả ta

**Reply by GiaHan06** [reply]
> kiểu kiếm bài thơ nói về cuộc sống sau khi covid được hạn chế hả ta

ý bà là những khó khăn sau covid hả

**Reply by HuynhLeBaoTran** [reply]
hay 2c nói về khó khăn trong vs sau covid

**Reply by HuynhLeBaoTran** [reply]
1c nói về tc cng

**Reply by GiaHan06** [reply]
tui thấy cái khó khăn gộp vô chung 1 luôn cũng được á
mà nãy tui mới nghĩ ra được thêm 1 ý nữa mà 5s sau tự nhiên quên ngang:< dạo này tui mất trí nhớ zữ lắm gòi

**Reply by chaudambaotram** [reply]
thui đừng bùn chắc xíu nữa nó quay về mà

**Comment by GiaHan06** [comment]
eeeeee chốt cái ròi phân công nè

**Reply by chaudambaotram** [reply]
lấy cái của bà nói i tui thấy đc mà

**Reply by HuynhLeBaoTran** [reply]
là khó khăn trong mùa covid-tình cảm con người-giá trị cuộc sống

**Reply by HuynhLeBaoTran** [reply]
phân công ii nè

**Reply by GiaHan06** [reply]
> là khó khăn trong mùa covid-tình cảm con người-giá trị cuộc sống

ổn ổn ha chốt ha

**Comment by GiaHan06** [comment]
_Nhiệm vụ:_
   - Mỗi người làm 1 cái dàn ý cho bài, ngắn gọn thoi ha mà phải nêu được những ý chính cần làm
   - Tìm 3 câu ca dao/tục ngữ/ thơ nói lên được quan điểm của mình trong bài làm

_Lưu ý:_ mình nên bắt đầu tìm cao dao tục ngữ từ bh nha, mấy câu đó thì nhiều nhma phải tìm câu hợp với bài làm thì sẽ khó....maybe, để có gì mà hong tìm ra thì còn sớm mình còn cách giải quyết chứ chiu chiu là toang nha
_Thời gian:_ từ ngày hôm nay 01/03 đến 10h sáng thứ 6 04/03

**Reply by HuynhLeBaoTran** [reply]
ucii nè bà^^

**Reply by GiaHan06** [reply]
nếu như xong trong ngày thứ 6 càng tốt nha tại qua thứ 7 là tuần 3 ròi thì phải á

**Reply by HienCaoPhuNhuan** [reply]
> nếu như xong trong ngày thứ 6 càng tốt nha tại qua thứ 7 là tuần 3 ròi thì phải á

Hạn thầy xem hoạt động thảo luận và Project mỗi tuần là hết thứ Tư

**Reply by GiaHan06** [reply]
> > nếu như xong trong ngày thứ 6 càng tốt nha tại qua thứ 7 là tuần 3 ròi thì phải á
> 
> Hạn thầy xem hoạt động thảo luận và Project mỗi tuần là hết thứ Tư

Dạ tại con tưởng hạn project phân công nhiệm vụ là hết thứ 4 còn thảo luận là hết tuần đó nên con hong biết. Con cảm ơn thầy vì đã nhắc nhở, con sẽ rút kinh nghiệm trong lần tới

**Comment by NguyenNgocQuynhAnh** [comment]
https://www.google.com.vn/amp/s/download.vn/nghi-luan-lien-quan-den-dich-covid-19-44144%3fmode=amp
đây nhó

**Comment by LTGHan** [comment]
Font nào cho đẹp đây mn :3?

**Comment by GiaHan06** [comment]
@HienCaoPhuNhuan dạ thầy ơi thầy, cái video record dung lượng lớn quá ạ tận 583 MB mà trên đây cho tải tối đa có 25 MB thoi nên là con đang tìm cách xử lí nó ạ T . T thầy thông cảm cho con nha thầy
Tụi con có up lên gg drive có gì thầy xem nha ạ
[https://drive.google.com/file/d/1dp5QPH5RFLxtgW7A18aBp8LJ1nyKIWBW](https://drive.google.com/file/d/1dp5QPH5RFLxtgW7A18aBp8LJ1nyKIWBW?fbclid=IwAR1uRmyX5kZTHNz_AxLIRoKKecnbd1V-dYKo-1e5IpmENoFhqToimvhvUBg)

**Reply by HienCaoPhuNhuan** [reply]
Khum sao, có cách xử lý là tốt
